mat64/symmetric.go

```diff
@@ -466,3 +469,32 @@ func (s *SymDense) GrowSquare(n int) Matrix {
 	v.cap = s.cap
 	return &v
 }
+
+// PowPSD computes a^pow where a is a positive symmetric definite matrix.
+//
+// PowPSD returns an error if the matrix is not PSD.
```

Comment:
"... if the matrix is not positive symmetric definite or no Eigendecomposition is possible."?